clarify docs generate steps (#2575)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/cloud-build-and-view-your-docs.md

4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.

> You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps. 

Reviewer: I suggest swapping these so we can add that the "generate docs on run" procedure was outlined in the previous steps. I also added the word "check box" and "selecting" so it's slightly easier to associate with the rows in the table.
```suggestion
You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes when adding the command to the run steps compared to configuring a job selecting the **Generate docs on run** checkbox (shown in previous steps). 
```

Reviewer: @mirnawong1 Do we have any docs that we can link to about adding to the list of commands in the job run steps? I couldn't find anything on a quick search but I must not have the right search terms.

Author: hey @runleonarun unfortunately we dont have anything specific, buuuuut @ernestoongaro and I are working on creating a document for this! see [here](https://www.notion.so/dbtlabs/Create-job-run-steps-docs-228d3e4c5a1f485592e1bd30e16f85ad)

Reviewer: Oh right! That will be soooo helpful. I didn't realize we had nothing about them. 😍

Review the following options and outcomes:

| Options | Outcomes |
|--------| ------- |

> | **1. Select checkbox** | Select the **Generate docs on run** checkbox to automatically generate updated project docs each time your job runs. If that particular step in your job fails, the job can still be successful if all subsequent steps are successful. |

Reviewer: They don't need to do both of these steps in this order, right? If not then I suggest removing step numbers which might be misleading.
```suggestion
| **Select checkbox** | Select the **Generate docs on run** checkbox to automatically generate updated project docs each time your job runs. If that particular step in your job fails, the job can still be successful if all subsequent steps are successful. |
```

> | **2. Add as a run step** | Add `dbt docs generate` to the list of commands in the job run steps, in whatever order you prefer. If that particular step in your job fails, the job will fail and all subsequent steps will be skipped.   |

Reviewer: Same comment as above!
```suggestion
| **Add as a run step** | Add `dbt docs generate` to the list of commands in the job run steps, in whatever order you prefer. If that particular step in your job fails, the job will fail and all subsequent steps will be skipped.   |
```

4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.

> You can also add `dbt docs generate` to the list of commands in the job run steps. Expect different outcomes and behaviors when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps.  

Author: ```suggestion
You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes and behaviors when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps.  
```

<!-- tabs for runs teps and command -->
<Tabs>

<TabItem value="checkbox" label="Generate docs on run checkbox">

> Selecting **Generate docs on run** means if the step your job fails, the job can still be successful. 

Author: ```suggestion
- Selecting **Generate docs on run** means if the step your job fails, the job can still be successful. 
```

> - Selecting **Generate docs on run** means if the step your job fails, the job can still be successful if all subsequent steps are successful. 

Author: ```suggestion
- Selecting **Generate docs on run** means if that particular step in your job fails, the job can still be successful if all subsequent steps are successful. 
```

</TabItem>

> <TabItem value="run_steps" label="`dbt docs generate` job run step">

Author: ```suggestion
<TabItem value="run_steps" label="<code>dbt docs generate</code> run step">
```

> <TabItem value="run_steps" label="<code>dbt docs generate</code> run step">

Author: ```suggestion
<TabItem value="run_steps" label="dbt docs generate run step">
```

> Adding `dbt docs generate` to the list of commands in the job run steps means if the the step in your job fails, the job will fail and all subsequent steps will be skipped.  

Author: ```suggestion
- Adding `dbt docs generate` to the list of commands in the job run steps means if the step in your job fails, the job will fail and all subsequent steps will be skipped.  
```

> - Adding `dbt docs generate` to the list of commands in the job run steps means if the step in your job fails, the job will fail and all subsequent steps will be skipped.  

Author: ```suggestion
- Adding `dbt docs generate` to the list of commands in the job run steps means if that particular step in your job fails, the job will fail and all subsequent steps will be skipped.  
```

4. Click **Save**. Proceed to [configure project documentation](#configure-project-documentation) so your project generates the documentation when this job runs.

> You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes and behaviors when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps.  

Reviewer: @mirnawong1 I feel like the TabItems might be adding unnecessary complexity for only two paragraphs. The more code we add, the harder the page is for a 1st time contributors to read, so reducing this might be a good idea. 
Consider this change:
```suggestion
You can also add `dbt docs generate` to the list of commands in the job run steps. However, you can expect different outcomes and behaviors when you configure a job using the **Generate docs on run** selection, compared to when you add the command to the run steps:

- Select the **Generate docs on run** checkbox to automatically generate updated project docs each time your job runs. If that particular step in your job fails, the job can still be successful if all subsequent steps are successful. 
- Add `dbt docs generate` to the list of commands in the job run steps, in whatever order you prefer. If that particular step in your job fails, the job will fail and all subsequent steps will be skipped.    

```

Author: thanks for that @runleonarun and totally understand! I've removed the tabitem code, however, i wonder if a table may be another good alternative?

i like the suggestions you provided, however, if i leave it as two bullets, i think it's not 100% clear that the two bullets are two different methods or options for the user. it reads as steps the user should take, when in fact the user can opt for the first bullet or the second, depending on their specific use case. 

I've taken the liberty to turn this into a table, what do you thinK?

> Review the different options and outcomes below:

Reviewer: ```suggestion
```

> <!-- tabs for runs teps and command -->

Reviewer: If you like the change above, then you can remove the following tabbItem content.
